apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.html

```diff
@@ -115,9 +115,10 @@ <h2>Flat</h2>
 
 <h2>Outlined</h2>
 <p>
-  An outlined variant of a card is shown only with a border around its edges and have no background.
-  Like the flat card, it is also without any shadow around its edges and the card itself is not
-  intended to be used for interactions (clickable, disclosure) as this also lacks proper affordance.
+  The outlined card variant is shown with a border around its edges and have no background. Like the
+  flat card, it is without any shadow around its edges and the card itself is not intended to be
```

Comment:
Consider simplifying:
```suggestion
  flat card it is not elevated as the card itself is not intended to be
```